src/libs/Middleware/Logging.js

```diff
@@ -103,7 +103,7 @@ function Logging(response, request) {
                     command: request.command,
                     message: error.message,
                     status: error.status,
-                });
+                }, false);
```

Comment:
Added this because the stack trace isn't necessary and was creating extra noise while debugging tests.